binding/Binding/SKTypeface.cs

```diff
@@ -108,6 +110,54 @@ public int CharsToGlyphs (IntPtr str, int strlen, SKEncoding encoding, out ushor
 				}
 			}
 		}
+
+		public string FamilyName {
+			get {
+				var r = SkiaApi.sk_typeface_get_family_name(Handle);
+				try {
+					var cstr = SkiaApi.sk_string_get_c_str(r);
+					var clen = SkiaApi.sk_string_get_size(r);
+					return Util.GetString(cstr, (int)clen, SKTextEncoding.Utf8); 
+				} finally {
+					SkiaApi.sk_string_destructor(r);
+				}
+			}
+		}
+
+		public UInt32[] GetTableTags()
+		{
+			int tableCount = SkiaApi.sk_typeface_count_tables(Handle);
+			UInt32[] result = new UInt32[tableCount];
+			int r = SkiaApi.sk_typeface_get_table_tags(Handle, result);
+			if (r == 0) {
+				throw new Exception("Unable to read the tables for the file.");
```

Comment:
@agallero I want to know what you think, but maybe we should have methods for each native operation. In this instance, we will have one for `sk_typeface_count_tables` and one for `sk_typeface_get_table_tags`. We can then have this one that calls those two. I am wanting to keep the APIs as close as possible.

Also, that exception... do we want it? Currently, exceptions are only thrown for invalid arguments. I am thinking we should do that here too. In this case, it might be better to return null.

If we have two separate methods, the user will get a `0` return, and I think this should match that with a null as `0` is an error and we can indicate that with `null` array.